### ISSUE TITLE
Allow themeing of query results grid

### ISSUE THREAD

**Issue by cryocaustik** [microsoft/azuredatastudio#2709]
currently we can edit the selected themes colors for specific elements in our user settings; suggestion is to allow editing the colors of query results grid elements as well. 

e.g. 
```json
{
    "workbench.colorCustomizations": {
        "[Default Dark Azure Data Studio]": {
            "queryResultsGrid.border": "#3b3b3b"
        }
    }
}
```

**Comment by palhal** [comment]
I am surprised that this seems impossible. The fixed color of "silver" is acceptable with light themes, but is way too obtrusive with dark themes. Dark mode is one of the main reasons I want to switch from SSMS.

I would like to turn the borders off (set them to transparent). The only way I can do this (to my knowledge) is by using jQuery in the console window: $('.slick-cell').css({ border: 0 }).

**Comment by endrx** [comment]
Yes, it's just awful. And still can't be changed through settings?
![image](https://user-images.githubusercontent.com/5373275/189352553-07e03bcf-69e7-40d1-b2d0-bae6cdbab1d9.png)

**Comment by mhokanson** [comment]
My issue was fixed by setting the "list.focusAndSelectionBackground" value for the theme.

**Comment by Harsh-Sugandhi-7** [comment]
> Is there a way to have the results color coded by data type? I would love it all the integers were shown in blue color and text in green, dates in orange etc ...

Exactly, what I came looking for...
